cmd/bosun/expr/funcs.go

```diff
@@ -104,24 +107,35 @@ var TSDB = map[string]parse.Func{
 		parse.TypeSeries,
 		tagQuery,
 		Band,
+		nil,
 	},
 	"change": {
 		[]parse.FuncType{parse.TypeString, parse.TypeString, parse.TypeString},
 		parse.TypeNumber,
 		tagQuery,
 		Change,
+		nil,
 	},
 	"count": {
 		[]parse.FuncType{parse.TypeString, parse.TypeString, parse.TypeString},
 		parse.TypeScalar,
 		nil,
 		Count,
+		nil,
```

Comment:
It feels a little wrong to add a new field to every one of these just because one or two functions use it. Are there any good alternatives?